.github/PULL_REQUEST_TEMPLATE.md

```
@@ -1,28 +1,127 @@
+- [ ] ⚙️ Config / CI / tooling change
 
-## Checklist:
+---
 
-- [ ] My code follows the style guidelines of this project
+## 🧪 How Has This Been Tested?
+
+<!-- Describe every test you ran. Be specific enough that a reviewer can reproduce it. -->
+
+### Test Environment
+| | |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)